src/core/lib/layer-manager.js

```diff
@@ -491,17 +512,16 @@ export default class LayerManager {
     const viewportChanged = !oldViewport || !viewport.equals(oldViewport);
 
     if (viewportChanged) {
-      Object.assign(this.oldContext, this.context);
+      log.log(4, 'Viewport', viewport)();
+
       this.context.viewport = viewport;
       this.context.viewportChanged = true;
```

Comment:
If `viewportChanged` is in `changeFlags`, do we still need this?

Reply:
Removed, nicely spotted!